src/shared/store/saga/products.ts

```diff
@@ -19,12 +19,15 @@ import {
   deleteProductSuccess,
 } from "../actions/products";
 import { PRODUCT_ACTIONS } from "../actionTypes/products";
+import { AxiosResponse } from "axios";
 
 function* fetchProductsWorker(): Generator {
   try {
     yield put(fetchProducts());
```

Comment:
This is pending?

I think prefer to make a craeteHttpAction with all types of SUCCESS, PENDING, FAILED
Combined them into one, all of them are related.

Reply:
i dont understand

Reply:
cant find a good way to do it, do you have any suggestions?